eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -85,7 +85,7 @@ stages:
                         inputs:
                           targetType: filePath
                           filePath: "eng/tools/publish-to-npm.ps1"
-                          arguments: '-pathToArtifacts $(Package.Archive) -accessLevel "public" -tag $(Tag) -additionalTag "$(AdditionalTag)" -registry ${{parameters.Registry}} -npmToken $(azure-sdk-npm-token)'
+                          arguments: '-pathToArtifacts $(Package.Archive) -accessLevel "public" -tag "$(Tag)" -additionalTag "$(AdditionalTag)" -registry ${{parameters.Registry}} -npmToken $(azure-sdk-npm-token)'
```

Comment:
That's correct. I am planning to change pipeline generator code to add these two variable for pipelines. Alternate option is to have variables defined in yaml with empty values. I am checking if run time variable has precedence over predefined variable in yaml or have check within yaml to set default value.

Reply:
Unfortunately you cannot define the empty value in yml and have it be overridden at queue time. It is a known limitation with Devops.

However given you are in a powershell context you can switch this to use an env variable instead. 
```suggestion
                          arguments: '-pathToArtifacts $(Package.Archive) -accessLevel "public" -tag $env:Tag -additionalTag "$(AdditionalTag)" -registry ${{parameters.Registry}} -npmToken $(azure-sdk-npm-token)'
```

When accessing the environment variable it will be empty if it isn't set but will contain the value of the queue variable if it is set. This has been a workaround for optional variables in other contexts.